fix(node): Update node tracing ESM imports. (#4876)
getsentry/sentry-docs: src/includes/getting-started-config/node.mdx

import * as Sentry from "@sentry/node";

// Importing @sentry/tracing patches the global hub for tracing to work.
import "@sentry/tracing";

> // If you want to use `Tracing` object in your project, you can use named imports such as:
> // import * as Tracing from "@sentry/tracing"

Reviewer: Nit:

```suggestion
// If you want to use `@sentry/tracing` in your project directly, use a named import instead:
// import * as SentryTracing from "@sentry/tracing"
```

(If you do this, I'd suggest switching the CJS to `SentryTracing` as well.)

Author: Done 👍
